third_party/terraform/tests/resource_compute_region_backend_service_test.go.erb

```diff
@@ -171,7 +171,6 @@ func TestAccComputeRegionBackendService_withConnectionDrainingAndUpdate(t *testi
 	})
 }
 
-<% unless version == 'ga' -%>
```

Comment:
Basic and Full tests here fail because of multiple RegionBackendService fields that are still beta-only.

Reply:
Thank you for catching this one. I did a quick diff of beta and v1 APIs:

```diff
--- regionBackendService.v1.json	2020-04-28 08:58:27.000000000 +0200
+++ regionBackendService.beta.json	2020-04-28 08:58:35.000000000 +0200
@@ -62,6 +62,10 @@
     "enable": boolean,
     "sampleRate": number
   },
+  "securitySettings": {
+    "authentication": string,
+    "subjectAltNames": [string]
+  },
   "localityLbPolicy": enum,
   "consistentHash": {
     "httpCookie": {
@@ -76,6 +80,10 @@
     "minimumRingSize": string
   },
   "circuitBreakers": {
+    "connectTimeout": {
+      "seconds": string,
+      "nanos": integer
+    },
     "maxRequestsPerConnection": integer,
     "maxConnections": integer,
     "maxPendingRequests": integer,
```

`circuitBreakers.connectTimeout` seems to be the only property that is beta-only (`securitySettings` is apparently not yet supported by magic-modules). I therefore only made the test that uses `circuitBreakers.connectTimeout` beta.

I ran the tests for the `google-provider` (didn't run beta) and they pass now:

```
TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test ./google -v -run=TestAccComputeRegionBackendService -timeout 240m -ldflags="-X=github.com/terraform-providers/terraform-provider-google/version.ProviderVersion=acc"
=== RUN   TestAccComputeRegionBackendService_regionBackendServiceBasicExample
=== PAUSE TestAccComputeRegionBackendService_regionBackendServiceBasicExample
=== RUN   TestAccComputeRegionBackendService_basic
=== PAUSE TestAccComputeRegionBackendService_basic
=== RUN   TestAccComputeRegionBackendService_withBackendInternal
=== PAUSE TestAccComputeRegionBackendService_withBackendInternal
=== RUN   TestAccComputeRegionBackendService_withBackendInternalManaged
=== PAUSE TestAccComputeRegionBackendService_withBackendInternalManaged
=== RUN   TestAccComputeRegionBackendService_withBackendMultiNic
=== PAUSE TestAccComputeRegionBackendService_withBackendMultiNic
=== RUN   TestAccComputeRegionBackendService_withConnectionDrainingAndUpdate
=== PAUSE TestAccComputeRegionBackendService_withConnectionDrainingAndUpdate
=== RUN   TestAccComputeRegionBackendService_ilbUpdateBasic
=== PAUSE TestAccComputeRegionBackendService_ilbUpdateBasic
=== CONT  TestAccComputeRegionBackendService_regionBackendServiceBasicExample
=== CONT  TestAccComputeRegionBackendService_withBackendMultiNic
=== CONT  TestAccComputeRegionBackendService_withBackendInternal
=== CONT  TestAccComputeRegionBackendService_basic
--- PASS: TestAccComputeRegionBackendService_regionBackendServiceBasicExample (51.36s)
=== CONT  TestAccComputeRegionBackendService_withBackendInternalManaged
--- PASS: TestAccComputeRegionBackendService_basic (78.24s)
=== CONT  TestAccComputeRegionBackendService_ilbUpdateBasic
--- PASS: TestAccComputeRegionBackendService_ilbUpdateBasic (77.79s)
=== CONT  TestAccComputeRegionBackendService_withConnectionDrainingAndUpdate
--- PASS: TestAccComputeRegionBackendService_withConnectionDrainingAndUpdate (68.68s)
--- PASS: TestAccComputeRegionBackendService_withBackendMultiNic (244.57s)
--- PASS: TestAccComputeRegionBackendService_withBackendInternal (275.21s)
--- PASS: TestAccComputeRegionBackendService_withBackendInternalManaged (314.16s)
PASS
ok  	github.com/terraform-providers/terraform-provider-google/google	366.149s
```

```
TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test ./google -v -run=TestAccComputeBackendService -timeout 240m -ldflags="-X=github.com/terraform-providers/terraform-provider-google/version.ProviderVersion=acc"
=== RUN   TestAccComputeBackendService_backendServiceBasicExample
=== PAUSE TestAccComputeBackendService_backendServiceBasicExample
=== RUN   TestAccComputeBackendServiceSignedUrlKey_basic
=== PAUSE TestAccComputeBackendServiceSignedUrlKey_basic
=== RUN   TestAccComputeBackendService_basic
=== PAUSE TestAccComputeBackendService_basic
=== RUN   TestAccComputeBackendService_withBackend
=== PAUSE TestAccComputeBackendService_withBackend
=== RUN   TestAccComputeBackendService_withBackendAndIAP
--- PASS: TestAccComputeBackendService_withBackendAndIAP (250.66s)
=== RUN   TestAccComputeBackendService_updatePreservesOptionalParameters
=== PAUSE TestAccComputeBackendService_updatePreservesOptionalParameters
=== RUN   TestAccComputeBackendService_withConnectionDraining
=== PAUSE TestAccComputeBackendService_withConnectionDraining
=== RUN   TestAccComputeBackendService_withConnectionDrainingAndUpdate
=== PAUSE TestAccComputeBackendService_withConnectionDrainingAndUpdate
=== RUN   TestAccComputeBackendService_withHttpsHealthCheck
=== PAUSE TestAccComputeBackendService_withHttpsHealthCheck
=== RUN   TestAccComputeBackendService_withCdnPolicy
=== PAUSE TestAccComputeBackendService_withCdnPolicy
=== RUN   TestAccComputeBackendService_withSecurityPolicy
=== PAUSE TestAccComputeBackendService_withSecurityPolicy
=== RUN   TestAccComputeBackendService_withCDNEnabled
=== PAUSE TestAccComputeBackendService_withCDNEnabled
=== RUN   TestAccComputeBackendService_withSessionAffinity
=== PAUSE TestAccComputeBackendService_withSessionAffinity
=== RUN   TestAccComputeBackendService_withAffinityCookieTtlSec
=== PAUSE TestAccComputeBackendService_withAffinityCookieTtlSec
=== RUN   TestAccComputeBackendService_withMaxConnections
=== PAUSE TestAccComputeBackendService_withMaxConnections
=== RUN   TestAccComputeBackendService_withMaxConnectionsPerInstance
=== PAUSE TestAccComputeBackendService_withMaxConnectionsPerInstance
=== RUN   TestAccComputeBackendService_withMaxRatePerEndpoint
=== PAUSE TestAccComputeBackendService_withMaxRatePerEndpoint
=== RUN   TestAccComputeBackendService_withMaxConnectionsPerEndpoint
=== PAUSE TestAccComputeBackendService_withMaxConnectionsPerEndpoint
=== RUN   TestAccComputeBackendService_withCustomHeaders
=== PAUSE TestAccComputeBackendService_withCustomHeaders
=== RUN   TestAccComputeBackendService_internalLoadBalancing
=== PAUSE TestAccComputeBackendService_internalLoadBalancing
=== RUN   TestAccComputeBackendService_withLogConfig
=== PAUSE TestAccComputeBackendService_withLogConfig
=== RUN   TestAccComputeBackendService_trafficDirectorUpdateBasic
=== PAUSE TestAccComputeBackendService_trafficDirectorUpdateBasic
=== CONT  TestAccComputeBackendService_backendServiceBasicExample
=== CONT  TestAccComputeBackendService_withSessionAffinity
=== CONT  TestAccComputeBackendService_trafficDirectorUpdateBasic
=== CONT  TestAccComputeBackendService_withLogConfig
--- PASS: TestAccComputeBackendService_backendServiceBasicExample (51.53s)
=== CONT  TestAccComputeBackendService_internalLoadBalancing
--- PASS: TestAccComputeBackendService_withLogConfig (65.84s)
=== CONT  TestAccComputeBackendService_withCustomHeaders
--- PASS: TestAccComputeBackendService_withSessionAffinity (66.48s)
=== CONT  TestAccComputeBackendService_withConnectionDrainingAndUpdate
--- PASS: TestAccComputeBackendService_trafficDirectorUpdateBasic (77.76s)
=== CONT  TestAccComputeBackendService_withCDNEnabled
--- PASS: TestAccComputeBackendService_withCDNEnabled (51.55s)
=== CONT  TestAccComputeBackendService_withSecurityPolicy
--- PASS: TestAccComputeBackendService_withCustomHeaders (65.38s)
=== CONT  TestAccComputeBackendService_withCdnPolicy
--- PASS: TestAccComputeBackendService_withConnectionDrainingAndUpdate (65.36s)
=== CONT  TestAccComputeBackendService_withMaxConnectionsPerInstance
--- PASS: TestAccComputeBackendService_withCdnPolicy (50.39s)
=== CONT  TestAccComputeBackendService_withMaxRatePerEndpoint
--- PASS: TestAccComputeBackendService_withSecurityPolicy (88.26s)
=== CONT  TestAccComputeBackendService_withMaxConnectionsPerEndpoint
--- PASS: TestAccComputeBackendService_internalLoadBalancing (297.67s)
=== CONT  TestAccComputeBackendService_withBackend
--- PASS: TestAccComputeBackendService_withMaxConnectionsPerInstance (258.86s)
=== CONT  TestAccComputeBackendService_withConnectionDraining
--- PASS: TestAccComputeBackendService_withConnectionDraining (51.23s)
=== CONT  TestAccComputeBackendService_updatePreservesOptionalParameters
--- PASS: TestAccComputeBackendService_updatePreservesOptionalParameters (65.78s)
=== CONT  TestAccComputeBackendService_withMaxConnections
--- PASS: TestAccComputeBackendService_withBackend (261.36s)
=== CONT  TestAccComputeBackendService_withAffinityCookieTtlSec
--- PASS: TestAccComputeBackendService_withAffinityCookieTtlSec (50.84s)
=== CONT  TestAccComputeBackendService_basic
--- PASS: TestAccComputeBackendService_withMaxRatePerEndpoint (533.88s)
=== CONT  TestAccComputeBackendServiceSignedUrlKey_basic
--- PASS: TestAccComputeBackendService_basic (77.01s)
=== CONT  TestAccComputeBackendService_withHttpsHealthCheck
--- PASS: TestAccComputeBackendService_withMaxConnectionsPerEndpoint (545.75s)
--- PASS: TestAccComputeBackendService_withMaxConnections (271.81s)
--- PASS: TestAccComputeBackendService_withHttpsHealthCheck (50.97s)
--- PASS: TestAccComputeBackendServiceSignedUrlKey_basic (76.16s)
PASS
ok  	github.com/terraform-providers/terraform-provider-google/google	1042.977s
```